examples/tweet_generator/tweet_generator.py

```diff
@@ -0,0 +1,180 @@
+import re
+import textwrap
+import os
+import sys
+from datetime import datetime
+from PIL import Image, ImageFont, ImageDraw, ImageOps
+from pincer import command, Client
+from pincer.commands import CommandArg, Description
+from pincer.objects import Message, Embed, MessageContext
+
+
+# you need to manually download the font files and put them into the folder
+# ./examples/tweet_generator/ to make the script works using this link:
+# https://fonts.google.com/share?selection.family=Noto%20Sans:wght@400;700
+if not all(font in os.listdir() for font in ["NotoSans-Regular.ttf", "NotoSans-Bold.ttf"]):
+    sys.exit()
+
+
+class Bot(Client):
+    @Client.event
+    async def on_ready(self):
+        print(
+            f"Started client on {self.bot}\n"
+            f"Registered commands: {', '.join(self.chat_commands)}"
+        )
+
+    @command(
+        description="to create fake tweets",
+    )
+    async def twitter(
+        self, ctx: MessageContext, content: CommandArg[str, Description["The content of the message"]]
+    ):
+        await ctx.interaction.ack()
+
+        for text_match, user_id in re.findall(
+            re.compile(r"(<@!(\d+)>)"), content
+        ):
+            content = content.replace(
+                text_match, f"@{await self.get_user(user_id)}"
+            )
+
+        if len(content) > 280:
+            return "A tweet can be at maximum 280 characters long"
+
+        # download the profile picture and convert it into Image object
+        avatar = (await ctx.author.user.get_avatar()).resize((128, 128))
+        avatar = circular_avatar(avatar)
+
+        # create the tweet by pasting the profile picture into a white image
+        tweet = trans_paste(
+            avatar,
+            Image.new("RGBA", (800, 250 + 50 * len(textwrap.wrap(content, 38))), (255, 255, 255)),
+            box=(15, 15),
+        )
+
+        # add the fonts
+        font_normal = ImageFont.truetype("NotoSans-Regular.ttf", 40)
+        font_small = ImageFont.truetype("NotoSans-Regular.ttf", 30)
+        font_bold = ImageFont.truetype("NotoSans-Bold.ttf", 40)
+
+        # write the name and username on the Image
+        draw = ImageDraw.Draw(tweet)
+        draw.text(
+            (180, 20), str(ctx.author.user),
+            fill=(0, 0, 0), font=font_bold
+        )
+        draw.text(
+            (180, 70),
+            f"@{ctx.author.user.username}",
+            fill=(120, 120, 120), font=font_normal,
+        )
```

Comment:
I would stay consistent with having 1 arg per line
```suggestion
        draw.text(
            (180, 20),
            str(ctx.author.user),
            fill=(0, 0, 0),
            font=font_bold
        )
        draw.text(
            (180, 70),
            f"@{ctx.author.user.username}",
            fill=(120, 120, 120),
            font=font_normal
        )
```